explainer-server/app/views/explainList.scala.html

```diff
@@ -46,12 +46,6 @@
                             </select>
                         </label>
                     </li>
-                    <li class="nav__item top-toolbar__item top-toolbar__item--no-spacing">
-                        <button class="top-toolbar__button"
-                           type="button" onclick="views.ExplainList().createNewExplainer()">

```

Comment:
Shouldn't `ExplainList` and `createNewExplainer` be removed too, then?

Reply:
At the end we will shutdown the entire project, so only preventing users to create new ones from the time being.

Reply:
oh yeah, that makes total sense 🚀